sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateDao.java

```diff
@@ -97,11 +101,19 @@ public PermissionTemplateDto selectByUuid(DbSession session, String templateUuid
     return mapper(session).selectByUuid(templateUuid);
   }
 
+  /**
+   * @deprecated does not support organizations. Should return group ids.
+   */
+  @Deprecated
   public List<PermissionTemplateDto> selectAll(DbSession session, String nameMatch) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)